src/game/client/swarm/vgui/asw_hud_chat.cpp

```diff
@@ -198,8 +198,8 @@ void CHudChat::MsgFunc_TextMsg( bf_read &msg )
 	char szString[2048];
 	int msg_dest = msg.ReadByte();
 
-	wchar_t szBuf[5][128];
-	wchar_t outputBuf[256];
+	wchar_t szBuf[5][512]; //Changed to 512 to increase buffer size for custom color.
+	wchar_t outputBuf[512]; //Changed to 512 to increase buffer size for custom color.
```

Comment:
```suggestion
	wchar_t szBuf[5][512];
	wchar_t outputBuf[512];
```
We've got public revision history and we're not expecting any upstream (Valve Alien Swarm) changes; we don't need comments describing changes.